model/host/db.go

```diff
@@ -467,6 +467,11 @@ func ById(id string) db.Q {
 	return db.Query(bson.D{{Key: IdKey, Value: id}})
 }
 
+func ByDistroIDRunning(distroID string) db.Q {
```

Comment:
Shouldn't this also have a filter to exclude spawn hosts?

Reply:
I added a filter for the user who started the host.

Reply:
I don't see it. Is it somewhere else?

Reply:
I forgot to push. Sorry!